Stratis.Bitcoin/Base/ChainHeadersBehavior.cs

```diff
@@ -7,319 +7,354 @@
 using NBitcoin.Protocol.Behaviors;
 using Stratis.Bitcoin.Connection;
 using Stratis.Bitcoin.Utilities;
+using Microsoft.Extensions.Logging;
 
 namespace Stratis.Bitcoin.Base
 {
+    /// <summary>
+    /// The Chain Behavior is responsible for keeping a ConcurrentChain up to date with the peer, it also responds to getheaders messages.
+    /// </summary>
+    public partial class ChainHeadersBehavior : NodeBehavior
+    {
+        /// <summary>Factory for creating loggers.</summary>
+        private readonly ILoggerFactory loggerFactory;
+
+        /// <summary>Instance logger.</summary>
+        private readonly ILogger logger;
+
+        /// <summary>Information about node's chain.</summary>
+        private readonly ChainState chainState;
+
+        /// <summary><c>true</c> if the chain should be kept in sync, <c>false</c> otherwise.</summary>
+        public bool CanSync { get; set; }
+
+        /// <summary><c>true</c> to sync the chain as headers come from the network, <c>false</c> not to sync automatically.</summary>
+        public bool AutoSync { get; set; }
+
+        /// <para>
+        /// The announced tip is accepted if it seems to be valid. Validation is only done on headers 
+        /// and so the announced tip may refer to invalid block.
+        /// </para>
+        /// </summary>
```

Comment:
Invalid ` </summary>`

Reply:
damn it seems i've deleted first part of the comment, will try to find it, but will definitely fix it somehow